src/docs/perf/rendering/shader.md

```diff
@@ -0,0 +1,171 @@
+---
+title: Reduce shader compilation jank on mobile
+short-title: Shader jank
+description: What is shader jank and how to minimize it.
+---
+
+If the animations on your mobile app appear to be janky,
+but only on the first run, you can _warm up_ the
+shader for the Skia Shader Language (SkSL) for a
```

Comment:
`shader for the Skia Shader Language (SkSL)` -> `shader captured in the Skia Shader Language (SkSL)`

Reply:
Yes, it's staged here:  https://sz-flutter.firebaseapp.com/docs/perf/rendering/shader

I'd like to go live tonight, not tomorrow.

Reply:
Oh, my bad. Somehow I thought that I saw "Wednesday" in some emails, maybe on another topic. CC @chinmaygarde @zanderso for the updated time window.

Reply:
The release goes live tomorrow at 8am, but this needs to be in place first.